.github/workflows/release.yml

```diff
@@ -122,3 +122,28 @@ jobs:
     - run: '[ -z "${GITHUB_REF%%refs/tags/*-pre*}" ] || ./script/packagecloud.rb'
       env:
         PACKAGECLOUD_TOKEN: ${{secrets.PACKAGECLOUD_TOKEN}}
+  build-docker-cross:
+    name: Build Cross Linux packages
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        arch: [arm64]
+        container: [debian_11]
+    - run: |
```

Comment:
I notice there two lines in the `build-docker` workflows from both this file and  `ci.yaml`, and also in the new `build-docker-cross` workflow in `ci.yaml`, which aren't present here:
```yaml
    - uses: actions/checkout@v1
    - uses: actions/setup-ruby@v1
```
Are these needed everywhere, or might they dropped everywhere, or is this workflow unique in some way in not needing them?

Reply:
Oh, no we definitely need those.  Good catch.